### PR TITLE
Increase caps on near, hold timing windows

### PR DESCRIPTION
**Pull request by ASleepyCat**
Should be inline with SDVX.

Prereq. PR: #458 to pass pipelines.

### CODE REVIEW

**Comment by ASleepyCat** [comment]
This should be fine to merge now that #458 has been merged and the build action has passed.

File: Beatmap/src/MapDatabase.cpp [hunk 1]
```diff
@@ -417,6 +416,14 @@ class MapDatabase_Impl
 				m_database.Exec("UPDATE Scores SET window_slam=84");
 				gotVersion = 19;
 			}
+			if (gotVersion == 19)
```
Comment (Drewol) [inline Beatmap/src/MapDatabase.cpp:419]: I don't really see the reason for updating the db and also altering past scores.

**Review by Drewol — CHANGES_REQUESTED** [review]

File: Beatmap/src/MapDatabase.cpp (hunk 1 above)
Comment (ASleepyCat) [reply Beatmap/src/MapDatabase.cpp:419]: I thought courses check the existing timing windows to see if the scores are eligible to be saved or not?

Comment (ASleepyCat) [reply Beatmap/src/MapDatabase.cpp:419]: Actually I think I see your point, nvm

Comment (ASleepyCat) [reply Beatmap/src/MapDatabase.cpp]: Removed.

**Review by Drewol — APPROVED** [review]

**Pull request merged**